code/modules/reagents/reagents/medicine.dm

```diff
@@ -327,7 +327,7 @@
 /datum/reagent/medicine/dexalinp
 	name = "Dexalin Plus"
 	id = "dexalinp"
-	description = "Dexalin Plus is used in the treatment of oxygen deprivation as well as muscle repair. It is highly effective."
+	description = "Dexalin Plus is used in the treatment of oxygen deprivation as well as muscle repair. Heals oxygen deprivation faster than Dexalin. Removes Lexorin from the bloodstream. Also slowly and slightly heals Muscles inside the body."
```

Comment:
Muscles is capitalized, unless the idea is to capitalize it to emphasize it’s a health „object“